lib/chef/knife/bootstrap.rb

```diff
@@ -375,9 +375,7 @@ def run
             exit 1
           end
 
-          client_builder.run
-
-          chef_vault_handler.run(node_name: config[:chef_node_name])
+          chef_vault_handler.run(client_builder.run[:client])
```

Comment:
Consider storing client and node in instance variables inside Chef::Knife::Bootstrap::Clientbuilder and pulling the client [and node] via an accessor.

```
client_builder.run
expect(client_builder.node).to eq(node)
expect(client_builder.client).to eq(client)
```